docs/tutorials/pytorch2onnx.md

```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
     --output-file checkpoints/resnet/resnet18_b16x8_cifar10.onnx \
     --dynamic-shape \
     --show \
+    --simplify \
     --verify \
 ```
 
 ## List of supported models exportable to ONNX
 
 The table below lists the models that are guaranteed to be exportable to ONNX and runnable in ONNX Runtime.
 
-|    Model     |                            Config                            | Note |
-| :----------: | :----------------------------------------------------------: | :--: |
-| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |      |
-|    ResNet    |          `configs/resnet/resnet18_b16x8_cifar10.py`          |      |
-|   ResNeXt    |     `configs/resnext/resnext50_32x4d_b32x8_imagenet.py`      |      |
-|  SE-ResNet   |       `configs/seresnet/seresnet50_b32x8_imagenet.py`        |      |
-| ShuffleNetV1 | `configs/shufflenet_v1/shufflenet_v1_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
-| ShuffleNetV2 | `configs/shufflenet_v2/shufflenet_v2_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
+|    Model     |                            Config                            | Batch Inference | Dynamic Shape | Note |
+| :----------: | :----------------------------------------------------------: | :-------------: | :-----------: | ---- |
+| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |        N        |       Y       |      |
```

Comment:
Batch inference should be OK for the exported ONNX models with ONNXRuntime backend. You could verify them.